run.py

```diff
@@ -1,36 +1,35 @@
 import argparse
 import json
 import os
-import re
 import google.auth
+from google.cloud import (bigquery, secretmanager)
+from google.cloud.secretmanager import SecretManagerServiceClient
 from mig_dx_api import (
     DX,
     DatasetSchema,
     Installation,
     SchemaProperty
 )
 from mig_dx_api._dataset import DatasetOperations
-from google.cloud import bigquery
 
-def get_target_installation(installations: list[Installation], target_installation_id: str = None) -> Installation:
+# TODO: name these according to how they have actually been named by DNC
+PRIVATE_KEY_SECRET_NAME='mig-portal-private-key'
+APP_ID_SECRET_NAME = 'mig-portal-application-id'
```

Comment:
Since this repo is public, would it be better to obscure these names and pass them in env variables? I know they shouldn't be sensitive on their own, but it does provide more information on the setup.

Reply:
Yeah, I was wondering about that too. Let me see what Chelsea thinks, since it adds a bit to the setup.